### PR TITLE
[WIP] Spanish translation.

### PR DESCRIPTION
**Pull request by C47D**
Hi,

Had a busy week at work last week but i had some free time today to continue the strings translations for Spanish language, this is clearly work in progress, if there's anyone who talk/understand Spanish please let me know.

There seems to be a merge issue as i haven't update my fork it in the last days, if you see something really wrong please let me know.

Carlos

### CODE REVIEW

**Comment by tannewt** [comment]
@C47D Thanks for the PR! You will need to update to a more recent version of master to get the string compression needed for the builds to work. Ping me on Discord if you need help with it.

I'm happy to get this merged before all strings are translated. Thanks!

**Comment by C47D** [comment]
Hi there @carlosperate , ¿como estas?.

No problem with the comments, actually i made the WIP available so others can work on it if they want (sort of what the french people did).

I will update my fork later tonight and work on your suggestions.

Carlos

File: locale/es.po [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-08-03 17:20-0700\n"
-"PO-Revision-Date: 2018-07-27 11:55-0700\n"
+"PO-Revision-Date: 2018-08-20 22:19-0500\n"
 "Last-Translator: \n"
 "Language-Team: \n"
 "Language: en_US\n"
```
[the remark this answers is not in the document]
Comment (C47D) [reply locale/es.po]: Yep, actually this was added automatically by the PO file editor i'm using at home, will see how to fix it.

File: locale/es.po [hunk 2]
```diff
@@ -19,207 +19,237 @@ msgstr ""
...
 "disable.\n"
 msgstr ""
+"Auto-recarga esta habilitado. Simplemente almacena los archivos vía USB para "
+"ejecutarlos o entra REPL para deshabilitar.\n"
 
 #: main.c:150
+#, fuzzy
 msgid "Running in safe mode! Auto-reload is off.\n"
-msgstr ""
+msgstr "Corriendo en modo seguro! Auto-recarga esta deshabilitado.\n"
```
[the remark this answers is not in the document]
Comment (C47D) [reply locale/es.po]: Same for me, i guess i did the literal translation, will fix.

File: locale/es.po [hunk 3]
```diff
@@ -885,17 +885,50 @@ msgstr ""
...
 msgid "function missing required keyword argument '%q'"
 msgstr ""
+||||||| merged common ancestors
+#: ports/atmel-samd/common-hal/analogio/AnalogOut.c:53
+msgid "AnalogOut not supported on given pin"
+msgstr ""
+=======
+#: ports/atmel-samd/common-hal/analogio/AnalogOut.c:53
+msgid "AnalogOut not supported on given pin"
+msgstr "AnalogOut no es soportado por el pin dado"
```
Comment (tannewt) [inline locale/es.po]: Looks like you missed some merge markers. Here you'll want to delete the empty "AnalogOut not supported on given pin" entry and the markers.

Comment (C47D) [reply locale/es.po]: Hi, fixed with 0aef043, please let me know if i missed something.

File: locale/es.po [hunk 4]
```diff
@@ -19,207 +19,237 @@ msgstr ""
 
 #: main.c:137
 msgid " output:\n"
-msgstr ""
+msgstr " salida:\n"
 
 #: main.c:148 main.c:221
+#, fuzzy
```
[the remark this answers is not in the document]
Comment (C47D) [reply locale/es.po]: Hi, thanks i will check that link, had a busy week until today :).

**Comment by tannewt** [comment]
Would you like me to polish this up so we can get it checked in?

**Comment by C47D** [comment]
Hi,

Yes if possible, but i would also like to know why the Travis build is failing, i checked the first failing test and it's marking the following error:
imagen

So if you can polish it it would be great, i'm not having as much free time as i would like to.

**Comment by tannewt** [comment]
That error happens when the po files are missing msgids that the code has. Its meant to ensure the files are always up to date with the code.

**Comment by tannewt** [comment]
I'm fixing this now and will push the fixes to your branch.

**Pull request merged**